docs/how-to/deploy/cloud-hosts/kubernetes-raw.mdx

```diff
@@ -44,4 +44,36 @@ To interact with the PDP **internally** (from inside the cluster), you can use t
 
 To interact with the PDP **externally** (from outside the cluster), you'll need to expose the service using a load balancer or an ingress of your choice.
 
+## Liveness, Readiness and Startup Probes
+
+The PDP exposes several endpoints that can be used for liveness, readiness and startup probes.
+
+### Liveness Probe
+
+The liveness probe is used to determine if the PDP is alive and healthy and should be restarted if it's not,
+it returns a 200 status code if the PDP is alive and healthy. The liveness probe endpoint is `/health`.
+An unhealthy PDP will return a 503 status code when The PDP failure rate is more than the configured threshold.
+<!-- An unhealthy PDP will return a 503 status code on one of the following scenarios: -->
+<!-- 1. The Policy Engine healthcheck returned an error or timed out. -->
+<!-- 2. The PDP failure rate is more than the configured threshold -->
+
+You can configure the failure threshold using environment variables:
+1. `PDP_OPA_CLIENT_FAILURE_THRESHOLD_PERCENTAGE` - The failure threshold percentage for the Policy Engine healthcheck. Default value is 10%.
+2. `PDP_OPA_CLIENT_FAILURE_THRESHOLD_INTERVAL` - The interval (in seconds) to calculate the failure threshold. Default value is 60 seconds.
+
+Default configuration will return 503 status code if at least 10% of the requests made to the PDP on the last 60 seconds failed.
+
+### Readiness Probe
+
+The readiness probe is used to determine if the PDP is ready to receive traffic,
+it returns a 200 status code if the PDP is ready to receive traffic. The readiness probe endpoint is `/healthy`.
+An unready PDP will return a 503 status code if the Policy Engine latest policy update failed.
+
+### Startup Probe
+
+The startup probe is used to determine if the PDP is ready to receive traffic on startup,
+it returns a 200 status code if the PDP finished the startup and is ready to receive traffic. The startup probe endpoint is `/ready`.
+An unready PDP will return 200 once the Policy Engine has finished to load the policy from the policy store ( Git repository ).
```

Comment:
This sentence is a bit misleading, letting you think that unready will return 200
I would say
Unready PDP will return <error code> and will recover once it ... and than return 200

Reply:
copy paste issue, fixed